hoodie-hadoop-mr/src/test/resources/sample1.avro

```diff
@@ -1,17 +1,23 @@
 {
```

Comment:
I'm using InputFormatTestUtil to simulate records (inserts and updates) and the code assumes a different schema than the one under SchemaTestUtils. I've added a new avro test file, under the hoodie-mr package, does that work ?